Update calculator.py

RandomNameGo1234 opened the pull request:
Added new function

RandomNameGo1234 committed:
Update calculator.py

Added new function

idi999 commented on calculator.py:
```diff
@@ -1,2 +1,7 @@
 def add(x, y):
   return x+y
+def subtract(x, y):
+  return x-y
```
can you make a variable and store the value of the subtraction to that variable then return the variable.

RandomNameGo1234 committed:
Update calculator.py

Made edits to file based on comments, result for subtraction function is now assigned to a variable

idi999 reviewed (commented):
Well done
